README.md

```diff
@@ -64,6 +64,21 @@ cp completion/kubie.fish ~/.config/fish/completions/
 
 Then reopen fish or source the file.
 
+
+#### Zsh
+
+Autocomplete with Zsh can either be installed by sourcing the file eg.:
+
+```bash
+source ./completion/kubie.zsh
+```
```

Comment:
AFAIK this will work for the current session ONLY. So it's not the solution or otherwise it should be properly noted.